RadarSDK/RadarAPIClient.m

```diff
@@ -49,29 +62,16 @@ - (void)getConfig {
     if (userId) {
         [queryString appendFormat:@"&userId=%@", userId];
     }
-    [queryString appendFormat:@"&deviceId=%@", [RadarUtils deviceId]];
-    [queryString appendFormat:@"&deviceType=%@", [RadarUtils deviceType]];
-    [queryString appendFormat:@"&deviceMake=%@", [RadarUtils deviceMake]];
```

Comment:
do we not need to duplicate certain params in the headers / body for backcompat, as we do with track?